packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol

```diff
@@ -33,40 +33,40 @@ contract CrossDomainMessengerLegacySpacer0 {
 contract CrossDomainMessengerLegacySpacer1 {
     /**
      * @custom:legacy
-     * @custom:spacer __gap
+     * @custom:spacer ContextUpgradable's __gap
      * @notice Spacer for backwards compatibility. Comes from OpenZeppelin
-     *         ContextUpgradable via OwnableUpgradeable.
+     *         ContextUpgradable.
```

Comment:
This is actually inherited from each of OwnableUpgradeable, PausableUpgradable and ReentrancyGuardUpgradeable.